site/src/pages/UsersPage/UsersPage.tsx

```diff
@@ -74,36 +78,47 @@ export const UsersPage: FC<{ children?: ReactNode }> = () => {
         status: option?.value,
       }),
   });
-  const authMethodsQuery = useQuery(authMethods());
-  const isLoading =
-    usersQuery.isLoading || rolesQuery.isLoading || authMethodsQuery.isLoading;
 
-  const [confirmSuspendUser, setConfirmSuspendUser] = useState<User>();
+  const [userToSuspend, setUserToSuspend] = useState<User>();
   const suspendUserMutation = useMutation(suspendUser(queryClient));
 
-  const [confirmActivateUser, setConfirmActivateUser] = useState<User>();
+  const [userToActivate, setUserToActivate] = useState<User>();
   const activateUserMutation = useMutation(activateUser(queryClient));
 
-  const [confirmDeleteUser, setConfirmDeleteUser] = useState<User>();
+  const [userToDelete, setUserToDelete] = useState<User>();
   const deleteUserMutation = useMutation(deleteUser(queryClient));
 
   const [confirmResetPassword, setConfirmResetPassword] = useState<{
     user: User;
     newPassword: string;
   }>();
-  const updatePasswordMutation = useMutation(updatePassword());
 
+  const updatePasswordMutation = useMutation(updatePassword());
   const updateRolesMutation = useMutation(updateRoles(queryClient));
 
+  // Indicates if oidc roles are synced from the oidc idp.
+  // Assign 'false' if unknown.
+  const oidcRoleSyncEnabled =
```

Comment:
Is this used somewhere?

Reply:
Honestly, I'm not completely sure. I didn't change the code or the comment, and I didn't dig deep enough into some of the components to see where it was actually used